packages/qulacs/tests/qulacs/circuit/test_convert_circuit.py

```diff
@@ -121,23 +124,30 @@ def test_convert_rotation_gate() -> None:
     # Check rotation angle sign
     c = np.cos(np.pi / 4)
     s = np.sin(np.pi / 4)
+
+    # the `.get_matrix()` method returns NDArray[complex128, _Shape]
+    # (_Shape should be the first argument)
```

Comment:
I think this is clearer:
```suggestion
    # We need to disable type check due to an error in qulacs type annotation
    # https://github.com/qulacs/qulacs/issues/537
```

And it may be sufficient to comment only once in one function.